Unreal Engine plugin documentation update (#6350)
getsentry/sentry-docs: src/platforms/unreal/setup-crashreporter.mdx

</Alert>

Alternatively, the endpoint can be added automatically to the _DefaultEngine.ini_ file
 by using the Sentry configuration window.

 ![The UE4 Crash Reporter settings](ue4-crash-reporter-settings.png)

> Navigate to the editor's menu **Project Settings > Plugins > Sentry**, set the Sentry _Unreal Engine Endpoint_ in the corresponding input field and click the **Update global settings** button to apply changes. Once the _Crash Reporter_ is not needed anymore the default endpoint value can be restored through the **Reset** button.
> 

Reviewer: ```suggestion
Navigate to the editor's menu **Project Settings > Plugins > Sentry**, set the Sentry _Unreal Engine Endpoint_ in the corresponding input field, and click the **Update global settings** button to apply changes. The default endpoint value can be restored by clicking the **Reset** button once the _Crash Reporter_ is no longer needed.

```
